pantheon-sessions.php

```diff
@@ -368,7 +474,15 @@ public function add_index() {
 
 		// Avoid errors by not attempting to add a column that already exists.
 		if ( ! empty( $key_existence ) ) {
-			$this->safe_output( __( 'ID column already exists and does not need to be added to the table.', 'wp-native-php-sessions' ), 'error' );
+			// If dealing with multisites, it's feasible that some may have the
+			// column and some may not, so don't stop execution if all that's
+			// wrong is the key exists on one.
+			if ( ! $multisite ) {
+				$type = 'error';
+			} else {
+				$type = 'log';
+			}
```

Comment:
I have it as an error for single sites because if a user thinks they should run the command when they should not on a single site, this is probably evidence of an issue of some sort. Additionally, further execution should be halted entirely, which throwing an error does.